### PR TITLE
Prepare new version of the charter

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

**Review by sudeepdi — COMMENTED** [review]
+1 Looks good to me

File: index.html [hunk 1]
```diff
@@ -377,20 +374,24 @@ <h3 id="w3c-coordination">
...
-              coordinate with this group to ensure the <a href=
-              "https://w3c.github.io/webrtc-nv-use-cases/">next version use
-              cases</a> can be implemented efficiently considering requirements
-              on the networks such as exchange of large files without
-              disruption to audio/video sessions, maintain a long-term
-              connection and seek to minimize power consumption. Secondarily,
-              explore implications of the new transport protocols such as the
-              QUIC.
+              Web of Things depends on networking technologies on the edge and
+              in the cloud when exposing interactions between Things, which can also include Edeg devices. The Web &
```
Comment (sudeepdi) [inline index.html:382]: Spelling error "Edeg". Change to "Edge"

**Review by sudeepdi — CHANGES_REQUESTED** [review]
Apart from one minor typo correction required (see comment inserted in doc), rest of the first draft looks good.

**Review by anssiko — APPROVED** [review]

**Review by sudeepdi — APPROVED** [review]
+1

**Pull request merged**